pyrtl/simulation.py

```diff
@@ -401,7 +401,8 @@ def step(self, provided_inputs):
         """
         # validate_inputs
         for wire, value in provided_inputs.items():
-            if value > wire.bitmask or value < 0:
+            if value > (self.block.get_wirevector_by_name(wire) if isinstance(wire, str)
```

Comment:
I would separate the finding of the wire to a different line.
It takes up a line of space anyways....